src/main/kotlin/uk/gov/dwp/dataworks/egress/services/impl/ManifestFileServiceImpl.kt

```diff
@@ -86,13 +86,14 @@ class ManifestFileServiceImpl(private val s3AsyncClient: S3AsyncClient,
     private suspend fun targetContentsAndRequest(specification: EgressSpecification, targetKey:String): Pair<ByteArray, PutObjectRequest> {
 
         val sourcePrefix = specification.sourcePrefix
-        val todaysDate = if (sourcePrefix.contains(TODAYS_YYYYMMDD_FORMATTED_DATE_PLACEHOLDER)) todaysDate("yyyyMMdd")
+        val todaysDate = if (specification.destinationPrefix.contains(TODAYS_YYYYMMDD_FORMATTED_DATE_PLACEHOLDER)) todaysDate("yyyyMMdd")
```

Comment:
How come we are using specification.destinationPrefix now instead of specification.sourcePrefix?

Reply:
The date format is changed only in destinationPrefix not on source, this was sort of bug.